ledger-core/virtual/object/object_migration_test.go

```diff
@@ -148,12 +162,18 @@ func TestSMObject_MigrationCreateStateReport_IfStateEmptyAndCountersSet(t *testi
 		LogMock.Return(smachine.Logger{}).
 		ShareMock.Return(smachine.NewUnboundSharedData(&report)).
 		PublishMock.Set(func(key interface{}, data interface{}) (b1 bool) {
-		assert.Equal(t, finalizedstate.BuildReportKey(report.Object), key)
 		assert.NotNil(t, data)
```

Comment:
extract that function too, before migrationCtx

Reply:
Fixed